teams/test-tools.toml

```diff
@@ -0,0 +1,22 @@
+name = "test-tools"
+subteam-of = "devtools"
+
+[people]
+leads = ["calebcartwright"]
+members = [
+    "calebcartwright",
+    "epage",
+    "Muscraft",
+    "thomcc",
+    "weihanglo",
+]
+
+[[github]]
+orgs = ["rust-lang"]
+
+[website]
+name = "Test tools team"
+description = "Defining strategy and associated tooling for supporting automated testing activities in the development workflow"
```

Comment:
This also focuses only on the tooling and not on the library side of things

From the RFC, we put our mission down as

> This team would be primarily focused on iterating on the test writing and analysis experience, cargo test, and enabling integration points and features for external tools like CI or IDEs.

Reply:
To make sure I fully understand, are you saying you'd rather reuse that sentence from the RFC verbatim, or that you feel this isn't a complete description and needs to be changed?

FWIW I don't think the one liner blurbs from the website trump the official scope articulated in places like the RFC and associated team repos. I've always looked at the governance pages of the website as extending to a much broader audience, including those who've potentially never seen (much less written) a line of Rust code, and I think it's helpful to try to generalize things as much as possible.

I personally view "tooling" as such a generalized word that would extend to frameworks, runners, libraries, etc. and not be exclusively scoped to command line executables/subcommands.

I'm fully open to any alternative phrasing though if there's any suggestions

Reply:
I was using that line as a point of comparison or idea generation.

For me, tooling is programs / automation and not the whole experience and I suspect a lot of people would be thrown off by the intent of the group if the name and/or description emphasizes tooling.

Reply:
That's fair. I'd like to avoid mentioning `cargo test`, `cargo bench`, etc. in this description though as I think that would also be a line of potential confusion given this would show on the same page with the description of the cargo team.

I'll think on this some more and try to come up with some proposals, and would encourage others to share any ideas as well

Reply:
As @thomcc says, this can always be changed later so it just has to be "good enough"